src/cascade/input_data/db/asdr.py

```diff
@@ -95,9 +83,23 @@ def _upload_asdr_data_to_tier_3(execution_context, cursor, model_version_id, asd
         "age_group_years_start": "age_lower",
         "age_group_years_end": "age_upper"
     })
-
     asdr_data = asdr_data.merge(age_group_data, how="left", on="age_group_id")
     asdr_data = asdr_data.where(pd.notnull(asdr_data), None)
+
+    ordered_cols = [
+        "year_id",
+        "location_id",
+        "sex_id",
+        "age_group_id",
+        "mean",
+        "upper",
+        "lower",
+        "age_upper",
+        "age_lower",
+    ]
+
+    asdr_data = asdr_data[ordered_cols]
+
```

Comment:
What if we used Pandas to_sql? Would that sort out the columns automatically for the upload? Or we could upload using the technique where you use todict() instead of tolist(). That way each row has keys and gets aligned correctly. Or make sqlalchemy metadata.

Walking back, would Pandas nicely order the upload for us?